package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "haxfred-slack-link-logging",
+  "version": "0.0.1",
+  "description": "Link logging module for haxfred and slack",
+  "repository": {
+    "url": "https://github.com/haxiom/haxfred-slack-link-logging"
+  },
+  "main": "./lib/haxfred-slack-link-logging",
+  "scripts": {
+    "test": "npm run lint && mocha",
+    "prepublish": "npm test && npm run build",
+    "build": "babel src --out-dir lib",
+    "lint": "eslint src test"
+  },
+  "author": {
+    "name": "Haxiom"
+  },
+  "contributors": [
+    {
+      "name": "Blade Barringer",
+      "email": "blade@crookedneighbor.com"
+    }
+  ],
+  "license": "GPL-2.0",
```

Comment:
I'm ok with either. There are things I like about both licenses but am not drawn to one or the other.